### ISSUE TITLE
TypeScript for the New Programmer, example obj.heigth error not shown

### ISSUE THREAD

**Issue by abhattal** [microsoft/TypeScript-Website#376]
On https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html in the "TypeScript: A Static Type Checker" section, the example with a misspelled object property that should show TypeScript picking up the error, does not show the error flagged by TypeScript, i.e.

```
Here’s the error TypeScript found:

const obj = { width: 10, height: 15 };
const area = obj.width * obj.heigth;
```

doesn't show any error.

**Comment by orta** [comment]
Ah yeah, that page probably hasn't had a run turning on twoslash for the code samples

**Comment by orta** [comment]
It does now, thanks!
